jetty-servlets/src/main/java/org/eclipse/jetty/servlets/DoSFilter.java

```diff
@@ -1235,7 +1235,7 @@ public OverLimit isRateExceeded(long now)
             }
 
             long rate = (now - last);
-            if (rate < 1000L)
+            if (TimeUnit.NANOSECONDS.toSeconds(rate) < 1L)
             {
                 return new Overage(Duration.ofMillis(rate), _maxRequestsPerSecond);
```

Comment:
Why is this still using `Duration.ofMillis` ?

Reply:
oops